### PR TITLE
Add $isoTimestamp to dynamic variables

### PR DESCRIPTION
**Pull request by atulbi**
Many APIs need timestamps to be send with the request and using ISO format is common to use. Currently, the dynamic variable only has `$timestamp` variable which provides the number of milliseconds since epoch.
Adding `$isoTimestamp` will provide the timestamp in ISO format at zero UTC.

Name: $isoTimestamp
Description: _The current ISO timestamp at zero UTC_

### CODE REVIEW

File: test/unit/dynamic-variables.test.js [hunk 1]
```diff
@@ -86,5 +86,14 @@ describe('Dynamic variable', function () {
...
             expect(directoryPath).to.not.be.null;
         });
+
+        it('$isoTimestamp returns a timestamp in ISO format', function () {
+            var isoTimestamp = dynamicVariables.$isoTimestamp.generator();
+
+            expect(isoTimestamp).to.be.a('string');
+            expect(isoTimestamp).to
+                // eslint-disable-next-line max-len
+                .match(/^([+-]?\d{4}(?!\d{2}\b))((-?)((0[1-9]|1[0-2])(\3([12]\d|0[1-9]|3[01]))?|W([0-4]\d|5[0-2])(-?[1-7])?|(00[1-9]|0[1-9]\d|[12]\d{2}|3([0-5]\d|6[1-6])))([T\s]((([01]\d|2[0-3])((:?)[0-5]\d)?|24:?00)([.,]\d+(?!:))?)?(\17[0-5]\d([.,]\d+)?)?([zZ]|([+-])([01]\d|2[0-3]):?([0-5]\d)?)?)?)?$/);
```
Comment (coditva) [inline test/unit/dynamic-variables.test.js:96]: Is there a way to simplify this regex? This is unreadable and we have no idea what/how it's testing.

File: lib/superstring/dynamic-variables.js [hunk 2]
```diff
@@ -602,6 +602,12 @@ var faker = require('faker/locale/en'),
         $randomSemver: {
             description: 'A random semantic version number',
             generator: faker.system.semver
+        },
+        $isoTimestamp: {
+            description: 'The current ISO timestamp at zero UTC',
```
Comment (coditva) [inline lib/superstring/dynamic-variables.js:607]: Have we gotten the variable name and copy approved?

File: CHANGELOG.yaml [hunk 3]
```diff
@@ -1,3 +1,7 @@
+master:
+  new features:
+    - Added $isoTimestamp to the dynamic variable list
```
Comment (coditva) [inline CHANGELOG.yaml:3]: Let's add the GitHub issue ID here as well: https://github.com/postmanlabs/postman-app-support/issues/553

File: lib/superstring/dynamic-variables.js [hunk 4]
```diff
@@ -602,6 +602,12 @@ var faker = require('faker/locale/en'),
         $randomSemver: {
             description: 'A random semantic version number',
             generator: faker.system.semver
+        },
+        $isoTimestamp: {
```
Comment (codenirvana) [inline lib/superstring/dynamic-variables.js:606]: Move it closer to `$timestamp`.

File: test/unit/dynamic-variables.test.js (hunk 1 above)
Comment (atulbi) [reply test/unit/dynamic-variables.test.js:96]: This covers many time-related corner cases. 
Let's update it with a simpler one. I will also add comments to it.

**Comment by IanKemp** [comment]
When can we expect an updated version of the Postman app with this change? It has been nearly 2 weeks now.

**Comment by atulbi** [comment]
App will be updated in 7.29 with these changes.
